cypress/support/pages/management/pageManagement.js

```diff
@@ -0,0 +1,159 @@
+
+'use strict'
+
+class Management {
+
+  static #chatToggleSwitch = '.rocketchat-switch'
```

Comment:
Imho, this should be placed below static #newSchoolAdminPageButton = '.btn-info' to be align with the application.